### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by gugaevkirill**

### CODE REVIEW

File: .swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata
Comment (gugaevkirill) [inline .swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata:1]: Woops, let's remove that file

File: Sources/CorrilySDK/DependencyManager/UserManager.swift [hunk 1]
```diff
@@ -44,7 +44,7 @@ public class UserManager {
     return parts.joined(separator: ":")
   }
   
-  public func setUser(userId: String? = nil, country: String? = nil) {
+    public func setUser(userId: String? = nil, country: String? = nil, notifyCorrilyBackEnd: Bool = true) {
```
Comment (gugaevkirill) [inline Sources/CorrilySDK/DependencyManager/UserManager.swift:47]: Related to:
https://www.notion.so/corrily/Mobile-SDK-8b45a52f5dc744b788ed24661becb495?pvs=4#05250fd4812e484aa32d7a474b6ad849

File: Sources/CorrilySDK/ViewModels/PaywallViewModel.swift [hunk 2]
```diff
@@ -31,7 +31,9 @@ public class PaywallViewModel: ObservableObject {
     do {
       let dto = PaywallDto(country: factory.user.country, userId: factory.user.userId, ip: factory.user.deviceId, paywallId: paywallId)
       let response = try await factory.api.getPaywall(dto)
-      DispatchQueue.main.async {
+      
+    // Shouldn't we use @MainActor instead or low-level DispatchQueue?
```
Comment (trangcongthanh) [inline Sources/CorrilySDK/ViewModels/PaywallViewModel.swift:35]: Yes, I changed to `@MainActor` in #2

File: Sources/CorrilySDK/Misc/Extensions/URLSession.swift [hunk 3]
```diff
@@ -11,7 +11,9 @@ import Foundation
 extension URLSession {
   func data(for request: URLRequest) async throws -> (Data, URLResponse) {
      try await withCheckedThrowingContinuation { continuation in
-       let task = self.dataTask(with: request, completionHandler: { data, response, error in
+       
+         // Shouldn't we use await dataTask here and remove completion handler?
```
Comment (trangcongthanh) [inline Sources/CorrilySDK/Misc/Extensions/URLSession.swift:15]: This extension provides async/await syntax for URLSession.data(for request: URLRequest) to support iOS 13 and 14. We can remove it when we target iOS 15+.

Comment (gugaevkirill) [reply Sources/CorrilySDK/Misc/Extensions/URLSession.swift:15]: aa, ok, let's keep supporting iOS 13

File: Sources/CorrilySDK/DependencyManager/UserManager.swift (hunk 1 above)
Comment (gugaevkirill) [reply Sources/CorrilySDK/DependencyManager/UserManager.swift:47]: Let's rename to `disableIdentificationRequest` or some other name, consistent w. README

**Comment by trangcongthanh** [comment]
@gugaevkirill I cherry-picked your update for readme to main. So I'll close to avoid conflict. Other tasks will done in separate PRs.

**Pull request closed without merging**